pom.xml

```diff
@@ -30,34 +30,34 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
you just change 
swapped places two plugin?Let's return the original state to this file and it will be removed from this PR

Reply:
It is still here) try just to copy the content of this file from our repository, and push it

Reply:
Actually, many people are having issues with pushing anything to GitHub right now :( I'll try again later